cms/static/js/spec/views/move_xblock_spec.js

```diff
@@ -336,9 +367,226 @@ define(['jquery', 'underscore', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpe
+                                        sourceDisplayName: displayName,
+                                        sourceLocator: locator,
+                                        parentSourceLocator: parentLocator,
+                                        targetIndex: sourceIndex
+                                    }
+                                }
+                            )
+                        ) !== -1
+                    )).toBeTruthy();
+                };
+
+                verifyNotificationStatus = function(requests, notificationSpy, notificationText, sourceIndex) {
+                    var sourceIndex = sourceIndex || 0;  // eslint-disable-line no-redeclare
+                    ViewHelpers.verifyNotificationShowing(notificationSpy, notificationText);
+                    AjaxHelpers.respondWithJson(requests, {
+                        move_source_locator: sourceLocator,
+                        parent_locator: sourceParentLocator,
+                        target_index: sourceIndex
+                    });
+                    ViewHelpers.verifyNotificationHidden(notificationSpy);
+                };
+
+                sendMoveXBlockRequest = function(requests, xblockLocator, targetIndex, sourceIndex) {
+                    var responseData,
+                        expectedData,
+                        sourceIndex = sourceIndex || 0; // eslint-disable-line no-redeclare
+
+                    responseData = expectedData = {
+                        move_source_locator: xblockLocator,
+                        parent_locator: modal.targetParentXBlockInfo.id
+                    };
+
+                    if (targetIndex !== undefined) {
+                        expectedData = _.extend(expectedData, {
+                            targetIndex: targetIndex
+                        });
+                    }
+
+                    // verify content of request
+                    AjaxHelpers.expectJsonRequest(requests, 'PATCH', '/xblock/', expectedData);
+
+                    // send the response
+                    AjaxHelpers.respondWithJson(requests, _.extend(responseData, {
+                        source_index: sourceIndex
+                    }));
+                };
+
+                moveXBlockWithSuccess = function(requests) {
+                    var sourceIndex = 0;
+                    // select a target item and click
+                    renderViews(courseOutline);
+                    _.each(_.range(3), function() {
+                        clickForwardButton(1);
+                    });
+                    modal.$el.find('.modal-actions .action-move').click();
+                    sendMoveXBlockRequest(requests, sourceLocator);
+                    expect(modal.movedAlertView).toBeDefined();
+                    verifyConfirmationFeedbackTitle(sourceDisplayName);
+                    verifyConfirmationFeedbackRedirectLink(sourceDisplayName);
+                    verifyConfirmationFeedbackUndoMoveAction(
```

Comment:
Can we rename above three methods and add Html in the end because all these methods are verifying rendered html.

`verifyConfirmationFeedbackTitle` -> `verifyConfirmationFeedbackTitleHtml`
`verifyConfirmationFeedbackRedirectLink` -> `verifyConfirmationFeedbackRedirectLinkHtml`
`verifyConfirmationFeedbackUndoMoveAction` -> `verifyConfirmationFeedbackUndoMoveActionHtml`